nx-dev/feature-ai/src/lib/feed-container.tsx

```diff
@@ -16,7 +17,8 @@ const assistantWelcome: ChatItem = {
 export function FeedContainer(): JSX.Element {
   const [error, setError] = useState<Error | null>(null);
   const [startedReply, setStartedReply] = useState(false);
-  const [sources, setSources] = useState<string[]>([]);
+  // Do we want to keep the sources in GA?
+  // const [sources, setSources] = useState<string[]>([]);
```

Comment:
Mm.. I don't think we need to. At least not for feedback.

Reply:
Ok! I will remove!

Reply:
Done